doc/whatsnew/2.12.rst

```diff
@@ -12,6 +12,10 @@ Summary -- Release highlights
 New checkers
 ============
 
+* Pyreverse - add output in mermaid-js format and html which is an mermaid js diagram with html boilerplate
+
+* Fix ``install graphiz`` message which isn't needed for puml output format.
+
 * Added ``missing-any-param-doc`` triggered when a function has neither parameter nor parameter type
```

Comment:
Can you please move this to ``doc/whatsnew/2.13.rst``, and put it below "Extensions" instead.